manifest/product-config.json

```diff
@@ -6,7 +6,7 @@
             "release_name": "Couchbase Sync Gateway",
             "production": true,
             "interval": 30,
-            "go_version": "1.22.2",
+            "go_version": "1.22.4",
```

Comment:
Let's change this version for 3.1.8 too in this file - this is read from `main` branch.

Reply:
Oops, I mean 3.1.9

Reply:
Updated version in "manifest/3.1.xml" to reflect this